lib/request/rack.rb

```diff
@@ -136,6 +136,7 @@ def if_modified_since
     accessor(:host,         Key.new('SERVER_NAME'))
     accessor(:query_string, Key.new('QUERY_STRING'))
     accessor(:content_type, Key.new('CONTENT_TYPE'), nil)
+    accessor(:cookies,      Key.new('HTTP_COOKIE'), EMPTY_STRING)
```

Comment:
> fair enough, then what about depending on snusnu/cookie and having #cookies do Cookie::Registry.coerce(header)

Yeah I like this. Can we somehow signal an "invalid header format", like having a `Request#valid?` that turns to `false` in case the `HTTP_COOKIE` value was invalidly formatted? I'd rather not process such a request as if there where no cookie present.

Might require more thoughts that are not in the scope of this PR.